001-update- rasppi.md

mcoyne1948 opened the pull request:
Fix broken download link in first paragraph of Manual Setup section

mcoyne1948 committed:
Update rasppi.md

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/NaomiProject/naomi-docs?pullRequest=60) <br/>All committers have signed the CLA.

AustinCasteel commented:
Thank you for catching the broken link. That being said the download link originally went to the raspbian download which I assume is now broken since raspbian is no longer an operating system since it was newly named RaspberryPi OS.

So the new link would actually be:
https://www.raspberrypi.org/software/operating-systems/

mcoyne1948 commented:
Thanks for the feedback. I am a newbie to Naomi but very enthusiastic 
(with lots of time on my hands). I have "volunteered" to help with 
documentation but am drinking from a fire hose right now learning about 
Naomi (playing with speech handler plugins), the documentation, git, 
markdown, etc. This submission was mostly a trial to see how to make an 
update.

Actually AaronC suggested you might be able help me with questions on 
audience(s), intent, orientation, structure, content, etc. I would also 
like to have someone to bounce ideas I have about things like structure 
and dealing with different user audiences.

Anyway if you can be of help I would appreciate hearing from you. I will 
continue to fumble along (slowly gaining understanding and competence as 
I go).

cheers, Malcolm

On 2021-01-15 11:52 a.m., Austin Casteel wrote:
>
> Thank you for catching the broken link. That being said the download 
> link originally went to the raspbian download which I assume is now 
> broken since raspbian is no longer an operating system since it was 
> newly named RaspberryPi OS.
>
> So the new link would actually be:
> https://www.raspberrypi.org/software/operating-systems/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/NaomiProject/naomi-docs/pull/60#issuecomment-761056064>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/APIEI5MTCZJEETMEAZV7LZDS2BXDDANCNFSM4WEIKCBA>.
>
-- 
Be Prepared

mcoyne1948 committed:
Update rasppi.md

Added the correct Raspberry Pi OS link to the download link in the Manual setup section
